lib/logitech_receiver/descriptors.py

```diff
@@ -191,6 +191,13 @@ def _D(name, codename=None, kind=None, wpid=None, protocol=None, registers=None,
 							_RS.hand_detection(),
 						],
 				)
+#'''
```

Comment:
Yes, of course. Sorry for not cleaning up after myself. I'll update the PR later today.